index.html

```diff
@@ -18,19 +18,24 @@
 
   <h1>Learn to code by watching others</h1>
 
-  See how experienced developers solve problems in real-time. Watching scripted tutorials is great, 
-  but understanding how developers think is invaluable. 
+  <p>See how experienced developers solve problems in real-time. Watching scripted tutorials is great, 
+  but understanding how developers think is invaluable. </p>
 
-  Try it free 7 days then $20/mo. thereafter
 
-  First Name 
-  Last Name 
-  Email Address
-  Password
+  <div>
+    <span>Try it free 7 days then $20/mo. thereafter</span>
+  </div>
 
-  Claim your free trial 
+<form>
+  <input type="text" placeholder="First Name"/> 
+  <input type="text" placeholder="Last Name"/> 
+  <input type="text" placeholder="Email Address"/>
+  <input type="text" placeholder="Password"/>
+  <button type="submit">Claim your free trial</button>
+</form>
+<span>By clicking the button, you are agreeing to our Terms and Services</span>
```

Comment:
I think we should contain all this markup inside a `div`, so we can style it better afterwards.

Reply:
How about a `section` instead?

Reply:
Sure, I think that would work!